app/models.py

```diff
@@ -30,38 +30,63 @@
 )
 from notifications_utils.timezones import convert_local_timezone_to_utc, convert_utc_to_local_timezone
 
-from app import DATETIME_FORMAT, encryption
+from app import encryption
+from app.constants import (
+    BRANDING_ORG,
+    DATETIME_FORMAT,
+    DELIVERY_STATUS_CALLBACK_TYPE,
+    EMAIL_TYPE,
+    EMAIL_AUTH_TYPE,
+    FIRETEXT_PROVIDER,
+    INVITE_PENDING,
+    INVITED_USER_STATUS_TYPES,
+    KEY_TYPE_NORMAL,
+    LETTER_TYPE,
+    MMG_PROVIDER,
+    MOBILE_TYPE,
+    NOTIFICATION_CREATED,
+    NOTIFICATION_DELIVERED,
+    NOTIFICATION_FAILED,
+    NOTIFICATION_RETURNED_LETTER,
+    NOTIFICATION_SENDING,
+    NOTIFICATION_STATUS_LETTER_ACCEPTED,
+    NOTIFICATION_STATUS_LETTER_RECEIVED,
+    NOTIFICATION_STATUS_TYPES,
+    NOTIFICATION_STATUS_TYPES_BILLABLE,
+    NOTIFICATION_STATUS_TYPES_COMPLETED,
+    NOTIFICATION_STATUS_TYPES_FAILED,
+    NOTIFICATION_TYPE,
+    PERMISSION_LIST,
+    PINPOINT_PROVIDER,
+    QUEUE_CHANNEL_TYPE,
+    SES_PROVIDER,
+    SMS_TYPE,
+    SNS_PROVIDER,
+    TEMPLATE_PROCESS_NORMAL,
+    TEMPLATE_TYPES,
+    WEBHOOK_CHANNEL_TYPE,
+    WHITELIST_RECIPIENT_TYPE,
+)
 from app.db import db
 from app.encryption import check_hash, hashpw
 from app.history_meta import Versioned
-from app.model import EMAIL_AUTH_TYPE, User
+from app.model import User
 from app.va.identifier import IdentifierType
 
-
-EMAIL_TYPE = 'email'
-LETTER_TYPE = 'letter'
-MOBILE_TYPE = 'mobile'
-PUSH_TYPE = 'push'
-SMS_TYPE = 'sms'
-
-TEMPLATE_TYPES = [SMS_TYPE, EMAIL_TYPE, LETTER_TYPE]
-
-template_types = db.Enum(*TEMPLATE_TYPES, name='template_type')
-
-NORMAL = 'normal'
-PRIORITY = 'priority'
-TEMPLATE_PROCESS_TYPE = [NORMAL, PRIORITY]
-
-DELIVERY_STATUS_CALLBACK_TYPE = 'delivery_status'
-COMPLAINT_CALLBACK_TYPE = 'complaint'
-INBOUND_SMS_CALLBACK_TYPE = 'inbound_sms'
-SERVICE_CALLBACK_TYPES = [DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE, INBOUND_SMS_CALLBACK_TYPE]
-
+# models.py only constants
+NOTIFICATION_STATUS_TYPES_NON_BILLABLE = list(set(NOTIFICATION_STATUS_TYPES) - set(NOTIFICATION_STATUS_TYPES_BILLABLE))
```

Comment:
Slightly more performant:
```suggestion
NOTIFICATION_STATUS_TYPES_NON_BILLABLE = tuple(frozenset(NOTIFICATION_STATUS_TYPES) - frozenset(NOTIFICATION_STATUS_TYPES_BILLABLE))
```

```python
>>> tuple(frozenset([1,2,3]) - frozenset([4,5,6]))
(1, 2, 3)
```

Reply:
Removed. It's not used.